go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -148,37 +147,33 @@ var influxAddr string = "http://meep-influxdb.default.svc.cluster.local:8086"
 var rc *redis.Connector
 var hostUrl *url.URL
 var sandboxName string
+var mepName string = defaultMepName
+var scopeOfLocality string = defaultScopeOfLocality
+var consumedLocalOnly bool = defaultConsumedLocalOnly
+var locality []string
 var basePath string
 var baseKey string
 var mutex sync.Mutex
 
 var gisAppClient *gisClient.APIClient
 var gisAppClientUrl string = "http://meep-gis-engine"
 
-//MEC011 section begin
 const serviceAppName = "Location"
 const serviceAppVersion = "2.1.1"
 
 var serviceAppInstanceId string
 
-var appEnablementClientUrl string = "http://meep-app-enablement"
-var appEnablementSupport bool = true
+var appEnablementUrl string
+var appEnablementEnabled bool = false
```

Comment:
Please don't initialize appEnablementEnabled as it suggest that this is the default mode which is not true.

The default mode is set by the environment variable.

Reply:
Okay, I will initialize it in the Init() function where the environment variable is read instead.